gallery/src/gallery-components/GalleryTiles.tsx

```diff
@@ -124,6 +153,27 @@ export const GalleryExampleTile: FunctionComponent<GalleryExampleTileProps> =
               <h2 className="nx-accordion__header-title">Example Code</h2>
             </NxAccordion.Header>
             {codeExampleElements}
+            <footer className="nx-footer">
+              <div className="nx-btn-bar">
+                {copyStatus === 'success' &&
+                  <>
+                    <NxFontAwesomeIcon icon={faCheckCircle}
+                                       title="Copied!"
+                                       color="var(--nx-swatch-green-25)"/>
```

Comment:
Please use CSS for this

Reply:
> Please use CSS for this

Stylesheet: c48f6ba7cc3f07b67a6fdca5b0e2aa4950ae4bdc
Component: 4922038b4e724fd2809aa2cb1858ce8ae89a7ff6